common/sagas/wallet.tsx

```diff
@@ -116,7 +116,9 @@ export function* unlockKeystore(action: UnlockKeystoreAction): SagaIterator {
   try {
     wallet = getKeystoreWallet(file, password);
   } catch (e) {
-    yield put(showNotification('danger', translate('ERROR_6')));
+    if (password !== '') {
+      yield put(showNotification('danger', translate('ERROR_6')));
+    }
```

Comment:
Even if the password is blank, it could throw for other reasons (Invalid file?) We should ideally check if the password is blank _and_ if it's a certain type of error.

Reply:
Woops! I should have seen that. Thanks for the feedback!

Reply:
Actually, testing this issue again without my changes, I see that there's no error at all for incorrect file uploads. I can upload jpgs to the keystore and it will give me an unlock button. Can't find this issue logged. Shall I open one?

Reply:
Yes, please do 👍